### ISSUE TITLE
Adjust hydroxyl hydrogen to have a small radius, requires more research

### ISSUE THREAD

**Issue by bannanc** [openforcefield/smirnoff99Frosst#61]
@davidlmobley @cbayly13 and I have discussed this off line, but the parameters for HX in parm@frosst were never added to SMIRNOFF99Frosst. 

SMARTS patterns from Patty:
`HX`: `[$([#1]-O-[#6]-,:[#7])]` AND `[#1$(*-[OX2]-[CX4]-[OX2$(*-[#1]),NX3+0])]`

We can write this as a single SMARTS pattern as 
`[#1$(*-[OX2]-[CX4]-[OX2$(*-[#1]),NX3+0]),#1$(*-O-[#6]-,:[#7]):1]`

Here's the line from the frcmod.parm_Frosst:
```
HX          0.6000  0.0157          Same param than H.  Sep 28 2004.  (HX is like HO but avoid electrostatic singularities in ketal and aminal because it has a small      Lennard Jones volume)
```

**Comment by bannanc** [comment]
@cbayly13 Suggested the following for the parameter on Slack:
" For the HX radius perhaps half the amide H, reducing from 0.6 down to 0.3, and to adjust the epsilon so there is no attraction (but no repulsion either) from outside the bonded oxygen's repulsive vdw radius."

**Comment by davidlmobley** [comment]
Yes, that was exactly my point. At some point I will want to try adding a separate very small hydroxyl radius to ALL hydroxyls (not just HX hydroxyls) and see how that affects results on our whole validation set.

**Comment by bannanc** [comment]
I see, that does sound like a better plan. 
I think it is probably worth having the issue on the tracker either way, I'll rename so it reflects the actual plan.

**Comment by jchodera** [comment]
> At some point I will want to try adding a separate very small hydroxyl radius to ALL hydroxyls (not just HX hydroxyls) and see how that affects results on our whole validation set.

Can we ensure that *all* atoms have `sigma > 0`? Are hydroxyl hydrogens the only ones with `sigma = 0` at this point?

**Comment by davidlmobley** [comment]
@jchodera - yes, that's the goal. It's only hydroxyl hydrogens at this point. And as I mentioned in the call, this is the AMBER legacy (ugh). I think this is a good opportunity to get rid of it.

**Comment by davidlmobley** [comment]
@bannanc - I have these changes basically ready to go (specifically, I have a new set of hydroxyl hydrogen parameters which are validated on densities and dielectric constants and ready to be incorporated). Are you ready for me to go ahead and change here and on `openforcefield`?

**Comment by bannanc** [comment]
@davidlmobley I think that is fine.

**Comment by bannanc** [comment]
This should have been closed by PR #74
